lorawan-device/src/region/fixed_channel_plans/mod.rs

```diff
@@ -263,6 +263,10 @@ impl<F: FixedChannelRegion> RegionHandler for FixedChannelPlan<F> {
         true
     }
 
+    fn channel_dl_update(&mut self, _: u8, _: u32) -> (bool, bool) {
```

Comment:
idk if unreachable makes sense here. I went with returning `Err(Error::CommandNotImplementedForRegion)`. Not much better but at least gives some context.

The spec says "SHALL NOT be implemented" for fixed channel plan regions so I guess silent ignore is what we want and that we already get with `if region.has_fixed_channel_plan()`